File: Java/Technology Fundamentals with Java/Exercises Java Code/src/MethodExercise/AddAndSubtract.java
package MethodExercise;

import java.util.Scanner;

public class AddAndSubtract {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int firstNum = Integer.parseInt(sc.nextLine());
        int secondNum = Integer.parseInt(sc.nextLine());
        int thirdNum = Integer.parseInt(sc.nextLine());
        int result = sum(firstNum, secondNum);
        result = subtract(result, thirdNum);

        System.out.println(result);
    }

    static int sum(int first, int second) {

        return first + second;
    }

    static int subtract(int first, int second) {

        return first - second;
    }
}

File: C#/Technology Fundamentals with C#/04. Methods - Functions - Exercise/05. Add and Subtract/AddAndSubtract.cs
using System;

public class AddAndSubtract
{
    public static void Main()
    {
        int firstNum = int.Parse(Console.ReadLine());
        int secondNum = int.Parse(Console.ReadLine());
        int thirdNum = int.Parse(Console.ReadLine());
        int sum = Sum(firstNum, secondNum);
        int substracedNumbers = Subtract(sum, thirdNum);
        Console.WriteLine(substracedNumbers);
    }

    private static int Subtract(int sum, int thirdNum)
    {
        int subtractedNumbers = sum - thirdNum;
        return subtractedNumbers;
    }

    private static int Sum(int firstNum, int secondNum)
    {
        int sum = firstNum + secondNum;
        return sum;
    }
}
